### PR TITLE
PhyX docs update to prevent blank landing page and mention PhysX version

### PR DESCRIPTION
**Pull request by lmbr-pip**
* Provides content to blank landing page for physics: https://docs.o3de.org/docs/user-guide/interactivity/physics/ found under https://docs.o3de.org/docs/user-guide/interactivity/ 
* Clarify that we are using PhysX 4 SDK as Nvidia landing page mentions both 5 and 4 and we should be clear about version.

Signed-off-by: Pip Potter <rppotter@amazon.com>

### Submission Checklist:

* [ x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Review by SergeyAMZN — APPROVED** [review]

File: content/docs/user-guide/interactivity/physics/_index.md
Comment (FiniteStateGit) [inline content/docs/user-guide/interactivity/physics/_index.md:2]:
```suggestion
linkTitle: Physics
title: Physics
description: Simulate physics, cloth, and destructible assets in Open 3D Engine (O3DE).
```

File: content/docs/user-guide/interactivity/physics/_index.md
Comment (FiniteStateGit) [inline content/docs/user-guide/interactivity/physics/_index.md:7]:
```suggestion
<!-- * [NIVDIA Blast](nvidia-blast) to provide fast, high-fidelity destruction simulation. -->
```
Blast docs are hidden for the time being.  `.\` isn't needed when stepping from index to a page in the common directory.

File: content/docs/user-guide/interactivity/physics/_index.md
Comment (FiniteStateGit) [inline content/docs/user-guide/interactivity/physics/_index.md:8]:
```suggestion
* [NIVDIA Cloth](nvidia-cloth) to support the creation of realistic cloth simulations.
```

File: content/docs/user-guide/interactivity/physics/_index.md
Comment (FiniteStateGit) [inline content/docs/user-guide/interactivity/physics/_index.md:9]:
```suggestion
* [NVIDIA PhysX](nvidia-physx) for realistic physical effects such as collision detection and rigid body dynamics simulation. 
```

File: content/docs/user-guide/interactivity/physics/_index.md
Comment (FiniteStateGit) [inline content/docs/user-guide/interactivity/physics/_index.md:5]:
```suggestion
**Open 3D Engine (O3DE)** supports physics simulation with the following components.
```
O3DE needed to be spelled out and bolded, but it looked odd with the possessive.

File: content/docs/user-guide/interactivity/physics/nvidia-physx/_index.md
Comment (FiniteStateGit) [inline content/docs/user-guide/interactivity/physics/nvidia-physx/_index.md:31]:
```suggestion
+ **[PhysX](/docs/user-guide/gems/reference/physics/nvidia/physx/)** - Provides integration for the [NVIDIA PhysX 4 SDK](https://developer.nvidia.com/physx-sdk) into O3DE. The integration provided includes a suite of components, configuration via the **O3DE Editor**, Script Canvas integration, **PhysX Visual Debugger** integration, and a simplified API abstraction layer for games.
```
Bolding tool names.

**Review by FiniteStateGit — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/interactivity/physics/_index.md
Comment (lmbr-pip) [reply content/docs/user-guide/interactivity/physics/_index.md:9]: Notice we have other references to blast https://o3de.org/docs/user-guide/interactivity/physics/nvidia-physx/ has links to "Simulated destruction with NVIDIA Blast" - I can clean those up if required

File: content/docs/user-guide/interactivity/physics/_index.md
Comment (lmbr-pip) [reply content/docs/user-guide/interactivity/physics/_index.md:7]: IMHO I would also avoid the use of the word component, unless you mean an actual O3DE component. Its an overloaded term.

Agree with changing O3DE to be spelt and bolded if thats the style. Will make that change.

**Review by micronAMZN — CHANGES_REQUESTED** [review]
I am ready to approve/merge this as is. This page will not be long lived as the PhysX docs and this section are on deck for improvement.

Just need the DCO fixed.

**Comment by lmbr-pip** [comment]
All my commits are signed, but it looks like DCO is failing because they are signed inconsistently for some reason. Unsure quite what has changed here:

Commit sha: 9694168, Author: Pip Potter, Committer: GitHub; Expected "Pip Potter 61438964+lmbr-pip@users.noreply.github.com", but got "rppotter rppotter@....".

**Review by FiniteStateGit — APPROVED** [review]
Approved.

**Review by willihay — CHANGES_REQUESTED** [review]
This PR now rewrites commits in the Script Canvas section that were merged yesterday. Since I'm continuing to work in this section, I'm concerned that if this PR is merged, it will lead to merge conflicts in my next PR.

**Comment by lmbr-pip** [comment]
> This PR now rewrites commits in the Script Canvas section that were merged yesterday. Since I'm continuing to work in this section, I'm concerned that if this PR is merged, it will lead to merge conflicts in my next PR.

Apologies, somehow the merge and DCO fixes have brought in extra files. Let me refresh the branch.

**Review by willihay — APPROVED** [review]

**Comment by lmbr-pip** [comment]
@micronAMZN Seems this is approved but I can't address your "requested" changes as I don't seem have a way to clear the flag without dismissing your review.

**Review by micronAMZN — APPROVED** [review]

**Pull request merged**
